gtfs/spec/en/reference.md

```diff
@@ -99,7 +99,7 @@ File: **Required**
 
 |  Field Name | Type | Required | Details |
 |  ------ | ------ | ------ | ------ |
-|  agency_id | ID | Optional | The **agency_id** field contains an ID that uniquely identifies a transit agency. A transit feed may represent data from more than one agency. This field is optional for transit feeds that only contain data for a single agency. |
+|  agency_id | ID | **Conditionally required** | The **agency_id** field contains an ID that uniquely identifies a transit agency. A transit feed may represent data from more than one agency. This field is optional for transit feeds that only contain data for a single agency. |
```

Comment:
For each of these fields, I suggest we follow the following format in the Details field at the end of the description:

>This field is required if..., otherwise it is optional.

Reply:
I've added the suggested language. Let me know what you think.